### PR TITLE
Added release note doc text guidance

### PR DESCRIPTION
**Pull request by ndeevy**
Hi folks,
Here is the suggested entry for how to write release note doc texts. It's quite a lengthy entry so needs feedback. The content is taken from this mojo article:
https://mojo.redhat.com/docs/DOC-1216602?sr=inbox&ru=50169
It includes a type of variable, so their format should follow the guidance that the Supplementary Style Guide eventually settles on.

### CODE REVIEW

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (JoanneOFlynn2018) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:120]:
```suggestion

```

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (JoanneOFlynn2018) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:121]:
```suggestion
```

**Comment by JoanneOFlynn2018** [comment]
I think it looks great Naomi, plenty of examples! The only tiny nit, I have is that all of the examples are RHOSP doc texts. Maybe it would be good to have an example from RHEL and OCP in there. What do you think?

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:11]: Should we cite this reference as _The IBM Style Guide_?

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:99]: I believe this should be two words: "life cycle".

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:104]: I believe this should be two words: "life cycle".

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:109]: I believe this should be two words: "life cycle".

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:113]: I believe this should be two words: "life cycle".

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: add a comma after "for example"

Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Should this be lowercase?

Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Delete "However" because you normally do not start a sentence with a conjunction.

Comment (leswilliams44) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Is this word necessary?

**Review by JoanneOFlynn2018 — DISMISSED** [review]

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy-redhat) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Done, thanks.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy-redhat) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Thanks Les, this little section is actually removed. It was something someone else wrote as part of the Release Monitor Playbook that I copied across accidentally at first. So it should be gone now if you look in the **Files changed** tab.
Thanks so much for your review! I implemented all your other changes.

**Comment by ndeevy-redhat** [comment]
> Maybe it would be good to have an example from RHEL and OCP in there. What do you think?
Thanks Joanne, I agree. I'll have to carve out some time to go searching for some suitable examples from other products so it'll have to wait a bit.
Thanks again :-]

**Comment by bergerhoffer** [comment]
@ndeevy Just wondering the status of this update, did you mean to close the PR?

**Comment by ndeevy** [comment]
> @ndeevy Just wondering the status of this update, did you mean to close the PR?

Thanks for the heads-up @bergerhoffer no, I did not mean to close it (new to PRs on GitHub). Also Joanne approved but wasn't sure if anyone else would like to approve then merge or should I just go ahead?

Thanx!

**Comment by bergerhoffer** [comment]
Thanks @ndeevy! I'll do a quick review since you requested me (sorry I meant to get to it sooner!)

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:14]: Since we use sentence case in CCS, I'd use that for this list of types, and in the corresponding headings below. Except "Technology Preview", which we standardized to always capitalize.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:11]: Suggest emphasizing the first instance of "doc text" here since it's a term we're introducing.

```
A _doc text_ is...
```

Comment (bergerhoffer) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:11]: Also:

* Errata Advisories -> Errata advisories
* Release Notes -> release notes

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:35]: Can you update the quotes in here to be normal keyboard quotes (for lack of a better term :laughing: )

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:71]: Minor nit, but I'd assume that `nova_wait_for_compute_service` should be in backticks.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:79]: We standardize to capitalize "Technology Preview", so this and the next examples should do so.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:65]: For all of these "Example doc texts" that precede, I'd suggest making them actual titles to the code block

```
.Example doc text
----
asdf
----
```

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:107]: Suggest putting each example doc text up with its general format.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:33]: Suggest putting each example doc text up with its general formatting.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:109]: I'm not sure if these examples using "will be" are appropriate.

If it's for the existing release's release notes, then it should be "is".

If it's for a future release, I think that we need to be careful about stating specifics. Typically I'd think that we'd normally want to avoid predicting what will happen in the future, as plans might change. @IngridT1 do you have any thoughts or know any rules on this?

**Comment by ndeevy** [comment]
Thanks for the excellent review @bergerhoffer !
I'll implement the changes next week.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:14]: These doc types are capitalized in Bugzilla, so I kept them in caps here. Would agree with keeping them this way in that case?
But by that logic they are gui elements really, in which case they should also be in bold font.  
WDYT @bergerhoffer ?

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:35]: Yes! But now that I look at the build again, these examples are already in code block formatting, so I don't think any quotes are necessary at all.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:71]: A worthy nit, thank you :)

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:109]: I wince at the sight of 'will' but from my understanding, warning users about future deprecation in release notes is the only real time it _is_ appropriate to use the future tense. Happy to be wrong though if @IngridT1  has any guidance :)

**Comment by leswilliams44** [comment]
@cbudz , What do I need to do to review/approve this PR?

**Comment by ndeevy** [comment]
> @cbudz , What do I need to do to review/approve this PR?

Hi @leswilliams44 , I still have some outstanding comments/questions on the entry that I think should be addressed before merging. I'd appreciate your input on them if you have any. Or if @bergerhoffer or @IngridT1 want to revisit the issues that'd be great too. Thanks!

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (IngridT1) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Sorry for the late comment, but I would remove "will", because it's understood here.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: @ndeevy Hmm I see what you're getting at. Though if I look at it in Bugzilla, then it should be "Enhancement" and "Bug Fix" singular. However, since this says that it could be for either Bugzilla or Jira (or any other tracking system really), I don't think that we need to match what Bugzilla uses, so I'd still go with sentence case in my opinion.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:30]: Since these are titles now, there shouldn't be a colon after (same for below ones as well)

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:9]: This whole guide is a collection of conventions, so I think we could just call this entry "Release notes doc texts"

I don't know if "doc texts" is an official/accepted term, but it does sound weird to me. But if others are okay with it, I'm fine with it as well, since we define the term here.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:11]: Release notes -> release notes

Comment (bergerhoffer) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc:11]: Engineering supplies -> Engineering typically supplies ?

I'm used to getting a draft, but I wasn't sure if that was always the case.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:21]: So this syntax (Enhancement::) causes the first line after it to be indented, but not others.

I'd either suggest making this a discrete heading:

```
[discrete]
==== Enhancement
```

Or using `+` between each of the items/examples under each type, so that they are indented properly with each type.

```
Enhancement::

Use present tense in one of the following formats:
+
----
This enhancement <present tense explanation>.
----
+
...
```

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:87]: This uses the same term syntax (Deprecation notice::) as the main "Deprecated Functionality" heading. This should be some other type of heading so that it can be known that it falls under "Deprecated Functionality". 

Also, the examples for each should be directly under the format example blocks.

Maybe it would also help to have the titles for the example code blocks say which type it's for. Like "Example deprecation notice doc text" and "Example removal notice doc text" and "Example enhancement doc text", etc.

**Review by bergerhoffer — COMMENTED** [review]
@ndeevy Just a few more thoughts/suggestions.

**Comment by bergerhoffer** [comment]
@ndeevy Are you able to take a look at this feedback?

**Comment by ndeevy** [comment]
> @ndeevy Are you able to take a look at this feedback?

Yep, will do. Have been swamped recently. Thanks for the nudge.

**Comment by bergerhoffer** [comment]
I'm going to mark this one as on hold for now. Let's revisit release notes as a whole after we get the first draft of the supplementary style guide out.

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Removed the 'will's. Thanks!

**Comment by ndeevy** [comment]
> I'm going to mark this one as on hold for now. Let's revisit release notes as a whole after we get the first draft of the supplementary style guide out.

Hi @bergerhoffer 
I implemented and resolved the review comments and think this entry is ready pending any more comments.  Do you think we could include it as its own entry: *Release notes doc texts* even though the bigger discussion about release notes as a whole is not tackled yet? I think that's a much broader conversation that will likely not be resolved for a long time. But I believe that this particular guidance has been fairly useful to release monitors, especially new ones. WDYT?

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (gregraka) [inline supplementary_style_guide/style_guidelines/specific-doc-types.adoc:118]: Hi Naomi (@ndeevy),
It looks like "Technology Preview features" is an orphaned topic and either needs to be developed or removed.
Thanks,
--Greg

**Review by gregraka — DISMISSED** [review]
Approved, pending removal of orphaned heading, "Technology Preview features."

File: supplementary_style_guide/style_guidelines/specific-doc-types.adoc
Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/specific-doc-types.adoc]: Thanks Greg, removed!

**Review by gregraka — APPROVED** [review]
LGTM! Thanks, Naomi! (@ndeevy)

**Comment by stoobie** [comment]
@redhat-documentation/ccs-style-council If you haven't already reviewed this and want to, then please do so. Otherwise, @cbudz @bergerhoffer It appears to me that this is ready to merge.

**Comment by cbudz** [comment]
Agreed, thanks @stoobie.

**Pull request merged**
